asyncpg/pool.py

```diff
@@ -798,14 +798,27 @@ def create_pool(dsn=None, *,
 
     Can be used either with an ``async with`` block:
 
+    .. code-block:: python
+
+        async with asyncpg.create_pool(user='postgres',
+                                       command_timeout=60) as pool:
+            await pool.fetch('SELECT 1')
+
+    Or to perform multiple operations on a single connection:
+
     .. code-block:: python
 
         async with asyncpg.create_pool(user='postgres',
                                        command_timeout=60) as pool:
             async with pool.acquire() as con:
+                await con.execute("""
```

Comment:
You'll need to change either docstring quotes to make this valid syntax.  Also, please indent. Thanks!